python/lsst/meas/extensions/psfex/psfex.py

```diff
@@ -158,16 +168,16 @@ def read_samples(prefs, set, filename, frmin, frmax, ext, next, catindex, contex
                     if key.naxis == 1 and n < key.naxisn[0]:
                         flux += n
                     else:
-                        print >> sys.stderr, "Not enough apertures for %s in catalogue %s: using first aperture" % \
-                            (prefs.getPhotfluxRkey(), filename)
+                        print("Not enough apertures for %s in catalogue %s: using first aperture" % \
+                            (prefs.getPhotfluxRkey(), filename), file=sys.stderr)
```

Comment:
because futurize doesn't do indenting fixes but autopep8 ran before you did futurize?